lib/internalexport.def

```diff
@@ -41,6 +41,9 @@ EXPORTS
     ktxTexture1_glTypeSize
     ktxTexture2_GetImageOffset
     ktxTexture2_calcLevelOffset
+    ktxTexture2_GetNumComponents
+    ktxTexture2_NeedsTranscoding
+    ktxTexture2_TranscodeBasis
```

Comment:
These 3 are already exported by the KTX_API decorations in ktx.h. Why do you need to add them here?

Reply:
True. I'll remove those and investigate which problem I (thought) this solved later.